yo1k/tic_tac_toe/core/game.py

```diff
@@ -216,22 +219,22 @@ def __advance_inround(self, state: State):
             assert False
 
     @staticmethod
-    def __occupy(state: State, cell: Cell):
+    def __occupy(state: State, cell: Cell) -> None:
         assert state.phase is Phase.INROUND
         mark = state.players[state.turn()].mark
         if state.board.get(cell) is None:
             state.board.set(cell, mark)
         else:
             assert False
```

Comment:
Replace
```python
if state.board.get(cell) is None:
    state.board.set(cell, mark)
else:
    assert False
```
with
```python
state.board.set(cell, mark)
```
and add the following assertion to `Board.set`:
```python
assert self.cells[cell.x][cell.y] is None
```

Reply:
Done.